packages/tools/kadena-cli/src/config/commands/configInit.ts

```diff
@@ -87,6 +95,15 @@ export const createConfigInitCommand: (
     logWalletInfo(created.words, wallet.filepath, key.publicKey);
 
     if (createAccount === 'true') {
+      // when --quiet is passed and account alias is not provided
+      // we will not create an account
+      if (!notEmpty(accountAlias)) {
+        log.error(
```

Comment:
Not sure about the error state, since the wallet did get created successfully. My initial feeling is that this should be a warning. (another reason commands should just do 1 thing... but that's a lost battle 😅)